src/itemStruct.ts

```diff
@@ -1,6 +1,59 @@
 import * as vscode from "vscode"
-import { getAutocompleteDump } from "./autocompleteDump"
-import { getApiDump, UNCREATABLE_TAGS } from "./dump"
+import { AutocompleteFunction, getAutocompleteDump } from "./autocompleteDump"
+import { ApiClass, ApiDump, getApiDump, UNCREATABLE_TAGS } from "./dump"
+
+const parameterClassFilter = (objectType: string, constraint: string) => {
+    return (klass: ApiClass): boolean => {
+        if (objectType === "Instance") {
+            if (constraint === "any") {
+                return true
+            } else if (constraint === "isScriptCreatable") {
+                const tags = klass.Tags
+                if (tags !== undefined) {
+                    for (const tag of tags) {
+                        if (UNCREATABLE_TAGS.has(tag)) {
+                            return false
+                        }
+                    }
+                }
+                return true
```

Comment:
`return klass.Tags?.every(tag => !UNCREATABLE_TAGS.has(tag))`

Reply:
Cant use optional chaining here because if tags is undefined, we want to return true
`return klass.Tags === undefined || klass.Tags.every(tag => !UNCREATABLE_TAGS.has(tag))` would be better